test/helpers/reducers.js

```diff
@@ -30,3 +30,22 @@ export function todosReverse(state = [], action) {
     return state;
   }
 }
+
+export function antipatternTodo(state = [], action) {
+  switch (action.type) {
+  case BAD_TODO:
+    action.boundDispatchFn();
+    return state;
+  default:
+    return state;
+  }
+}
+
+export function antipatternTodoWithError(state = [], action) {
```

Comment:
I guess this one is just `throwingTodo`? Not really fair to call it `antipattern*` because it doesn't `dispatch` itself.